pingrid.py

```diff
@@ -655,32 +655,47 @@ def ring_shapely_to_leaflet(ring):
 # Flask utils
 
 
-class InvalidRequest(Exception):
-    def __init__(self, message):
-        super().__init__()
```

Comment:
Because we didn't pass the message along to the superclass constructor, that message didn't get logged along with the stack trace.